engine/context/ecs-integration.md

```diff
@@ -73,6 +75,56 @@ their state using the `docker compose ps` command.
 - You can view logs from containers that are part of the Compose application
 using the `docker compose logs` command.
 
+## Rolling update
+
+Your ECS services are created with rolling update configuration. As you run

```

Comment:
Can we start with a simple statement here, something like:
to update your application without interrupting production flow you can simple use Compose up.

Reply:
updated